sass/_theme_variables.sass

```
@@ -51,6 +51,7 @@ $nav-link-color:                      $blue
 $nav-link-color-visited:              $purple
 $nav-link-color-hover:                lighten($nav-link-color, 6%) !default
 $nav-link-color-alt:                  hsl(33, 100%, 51%)
+$nav-caption:                         $blue
```

This color is a little overbearing, maybe desaturate it a bit?
```suggestion
$nav-caption:                         desaturate($blue, 15%)
```